impeller/renderer/backend/vulkan/render_pass_cache_unittests.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/testing/testing.h"
+#include "gtest/gtest.h"
+#include "impeller/playground/playground_test.h"
+#include "impeller/renderer/backend/vulkan/texture_vk.h"
+#include "impeller/renderer/render_target.h"
+
+namespace impeller {
+namespace testing {
+
+using RendererTest = PlaygroundTest;
+
+TEST_P(RendererTest, CachesRenderPassAndFramebuffer) {
+  if (GetBackend() != PlaygroundBackend::kVulkan) {
+    GTEST_SKIP() << "Test only applies to Vulkan";
+  }
+
+  auto allocator = std::make_shared<RenderTargetAllocator>(
+      GetContext()->GetResourceAllocator());
+
+  auto render_target = RenderTarget::CreateOffscreenMSAA(
+      *GetContext(), *allocator, {100, 100}, 1);
+  auto resolve_texture =
+      render_target.GetColorAttachments().find(0u)->second.resolve_texture;
+  auto& texture_vk = TextureVK::Cast(*resolve_texture);
+
```

Comment:
This test appears to run with validations locally. There were no validation errors I believe